share/spice/meetup/meetup.js

```diff
@@ -0,0 +1,45 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_meetup = function(api_result){
+
+        if (!api_result || api_result.error || !api_result.results) {
```

Comment:
Looks like it should be `errors` instead of just `error` that we should watch out for http://www.meetup.com/meetup_api/docs/:

![screen shot 2015-09-17 at 3 13 05 pm](https://cloud.githubusercontent.com/assets/81969/9943320/a86de63e-5d4e-11e5-8078-25907b093081.png)